commit/chainfee/outcome.go

```diff
@@ -214,22 +217,29 @@ func (p *processor) getGasPricesToUpdate(
 	lggr logger.Logger,
 	currentChainUSDFees map[cciptypes.ChainSelector]ComponentsUSDPrices,
 	latestUpdates map[cciptypes.ChainSelector]Update,
-	obsTimestamp time.Time,
+	consensusTimestamp time.Time,
 ) []cciptypes.GasPriceChain {
 	var gasPrices []cciptypes.GasPriceChain
 
 	for chain, currentChainFee := range currentChainUSDFees {
 		chainCfg, err := p.homeChain.GetChainConfig(chain)
-
-		feeConfig := chainCfg.Config
 		if err != nil {
 			lggr.Errorw("error getting chain config", "chain", chain, "err", err)
 			continue
 		}
```

Comment:
Moved `if err != nil` right after the `GetChainConfig` call for more idiomatic code.